charts/kubearchive/templates/role_binding.yaml

```diff
@@ -28,3 +28,16 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
   name: {{ .Values.apiServer.name }}
+---
```

Comment:
Giving a second thought, adding a SA with cluster-wide `view` in a non-specified namespace feels like a security issue. Remove it and we can discuss how to add it in a separate issue.

Reply:
Note that in a production cluster, the users will have different permissions and they will be probably linked to an oauth server. This is for testing purposes and there are no users deployed by default in the kind cluster apart from the admin one so we need to include one with some "get" privileges.

We could add a specific role for that or use the "view" one that is given by default in the cluster.

As the helm chart is used for deploying a testing environment I don't think that we should remove this and wait for another issue to be done because that will make the REST API unusable. I prefer to keep this and if we want to provide a more restrictive role to test the API in a different issue, remove this in that moment.

Reply:
If the helm chart is just scoped for development then its fine to keep it as is.